Manito/Manito/Screens/Detail-Ing/DetailngCodebaseViewController.swift

```diff
@@ -0,0 +1,378 @@
+//
+//  DetailngCodebaseViewController.swift
+//  Manito
+//
+//  Created by creohwan on 2022/11/03.
+//
+
+import UIKit
+import SnapKit
+
+class DetailingCodebaseViewController: BaseViewController {
```

Comment:
p4;

현재 클래스명에 codebase가 들어가는 이유가 스토리보드로 만들어진 viewController가 있어서라고 생각이 듭니다.
추후에 스토리보드가 삭제되고 나선 클래스명에 codebase란 단어를 빼야할 것 같아서, FIXME를 활용해 메모해 두시면 좋을 것 같습니다 !

```suggestion
// FIXME: 스토리보드 삭제 후 클래스명 변경
class DetailingCodebaseViewController: BaseViewController {
```

Reply:
22

Reply:
> p4;
> 
> 현재 클래스명에 codebase가 들어가는 이유가 스토리보드로 만들어진 viewController가 있어서라고 생각이 듭니다. 추후에 스토리보드가 삭제되고 나선 클래스명에 codebase란 단어를 빼야할 것 같아서, FIXME를 활용해 메모해 두시면 좋을 것 같습니다 !

79d7189 넵 감사합니다~